src/Node.php

```diff
@@ -312,35 +310,11 @@ protected function getChildrenKvPairs() {
     }
 
     public function jsonSerialize() {
-        $kindName = self::getNodeKindNameFromValue($this->getKind());
+        $kindName = self::getNodeKindName();
```

Comment:
Good catch - it doesn't make much of a functional difference at the moment, but might help if we ever decide to override `getNodeKindName`